superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -210,13 +210,6 @@ export default function DataSourcePanel({
     setInputValue('');
   }, [columns, datasource, metrics]);
 
-  const metricSlice = showAllMetrics
```

Comment:
Just confirming -- this landed somewhere else from upstream, right?

Reply:
I believe so, I need to do some testing but it was marked as unused after merging